DQMOffline/Trigger/plugins/MssmHbbBtagTriggerMonitor.cc

```diff
@@ -0,0 +1,175 @@
+#include <math.h>
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+
+
+#include "DQMOffline/Trigger/plugins/MssmHbbBtagTriggerMonitor.h"
+
+#include "TLorentzVector.h"
+
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+MssmHbbBtagTriggerMonitor::MssmHbbBtagTriggerMonitor( const edm::ParameterSet& iConfig )
+{
+  folderName_             = iConfig.getParameter<std::string>("dirname");
+  processname_            = iConfig.getParameter<std::string>("processname");
+  pathname_               = iConfig.getParameter<std::string>("pathname");
+  triggerobjbtag_         = iConfig.getParameter<std::string>("triggerobjbtag");
+  triggerSummaryLabel_    = iConfig.getParameter<edm::InputTag>("triggerSummary");
```

Comment:
do you really need it as global variable ?
the same applies to the other InputTag